### PR TITLE
bitmap reuse

### PR DESCRIPTION
**Pull request by nwadams**
inBitmap image reuse

### CODE REVIEW

File: src/com/android/volley/toolbox/NetworkImageView.java [hunk 1]
```diff
@@ -184,11 +186,11 @@ public void run() {
                             return;
                         }
 
-                        if (response.getBitmap() != null) {
+                        if (response.getDrawable() != null) {
```
Comment (lxdvs) [inline src/com/android/volley/toolbox/NetworkImageView.java:189]: can you override response.getBitmap() to return the internal BitmapDrawable's bitmap?

**Comment by lxdvs** [comment]
I wonder if there is any way we can minimize the number of lines changed by more aggressive subclassing

File: src/com/android/volley/toolbox/NetworkImageView.java (hunk 1 above)
Comment (nwadams) [inline src/com/android/volley/toolbox/NetworkImageView.java:189]: I need to surface the CacheableBitmapDrawable so I can manage its state in the NetworkImageView

**Comment by nwadams** [comment]
lets talk about a way we can do some more aggressive subclassing. If you have suggestions let me know.

**Comment by lxdvs** [comment]
I suggest we hold off on this until tomorrow / post-branchcut. Deal?

**Comment by nwadams** [comment]
+1 on holding

**Pull request merged**